ext/standard/tests/file/fputcsv_variation17.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+fputcsv() with user provided eol
+--FILE--
+<?php
+$data = [
+    ['aaa', 'bbb', 'ccc', 'dddd'],
+    ['123', '456', '789'],
+    ['"aaa"', '"bbb"'],
+];
+
+$eol_chars = ['||', '|', '\n', "\n"];
```

Comment:
```suggestion
$eol_chars = ['||', '|', '\n', "\n", "\0"];
```
To test for a null byte, but the output is going to be a bit annoying due to that, might want to make it as another test to limit the number of possible "binary file" which prevents seeing a diff.